python/lsst/sims/catUtils/mixins/VariabilityMixin.py

```diff
@@ -2,13 +2,24 @@
 import linecache
```

Comment:
It does not look like this package is being used. So maybe the import can be dropped.